src/Sdk/WebApi/WebApi/Contracts.cs

```diff
@@ -126,6 +127,46 @@ public class CreateMetadataResponse
         public bool Ok;
     }
 
+    [DataContract]
+    [JsonObject(NamingStrategyType = typeof(SnakeCaseNamingStrategy))]
+    public class StepsUpdateRequest
+    {
+        [DataMember]
+        public IEnumerable<Step> Steps;
+        [DataMember]
+        public long ChangeOrder;
+        [DataMember]
+        public string WorkflowJobRunBackendId;
+        [DataMember]
+        public string WorkflowRunBackendId;
+    }
+
+    [DataContract]
+    [JsonObject(NamingStrategyType = typeof(SnakeCaseNamingStrategy))]
+    public class Step
+    {
+        [DataMember]
+        public string ExternalId;
+        [DataMember]
+        public int Number;
+        [DataMember]
+        public string Name;
+        [DataMember]
+        public Status Status;
+        [DataMember]
+        public string StartedAt;
+        [DataMember]
+        public string CompletedAt;
+    }
+
+    public enum Status
+    {
+        StatusUnknown = 0,
+        StatusInProgress = 3,
+        StatusPending = 5,
+        StatusCompleted = 6
```

Comment:
why not 0, 1, 2, 3?

Reply:
Those numbers are defined elsewhere.  I removed the ones that are not applicable to runner.